### ISSUE TITLE
Dockerfile does not work with docker version 24

### ISSUE THREAD

**Issue by stefanhaerter** [RotherOSS/otobo#3782]
Docker version 24.0.7, build 24.0.7-0ubuntu2~22.04.1

Apparently, this docker version does not like the HEREDOC syntax like here:

https://github.com/RotherOSS/otobo/blob/1663607b16a632f96a723f26e87992e6174bdc07/otobo.web.dockerfile#L106-L118

**Comment by bschmalhofer** [comment]
Changing the Dockerfiles to not using here documents is no problem. But I think that we should first discuss the required versions of Docker. We could also require different versions for running and for building Docker images.

**Comment by stefanhaerter** [comment]
According to some blog posts, heredoc support was added during 2021, which surprised me a bit because the docker version which I run seems to have been released in 2023. But I couldn't figure out which versions exactly have heredoc support.

**Comment by bschmalhofer** [comment]
Yes, this is a bit confusing. Another thing is that the builds on Docker Hub and GitHub are apparently building with buildkit. I think that the same should be done when building locally.
See https://docs.docker.com/build/buildkit/#getting-started . But still more confusion ensues. From reading the documentation I would have assumed that buildkit is used per default when running Docker 23 or later.

@stefanhaerter : Do you have special setup in your _/etc/docker/daemon.json_ file ?

**Comment by stefanhaerter** [comment]
I have the folder /etc/docker on my system, but it is empty.

**Comment by stefanhaerter** [comment]
In the meantime, I built images locally several times and the problem did not show up again. Suggesting to close the issue.

**Comment by bschmalhofer** [comment]
Thanks for testing. Being able to use HERE documents makes the Dockerfiles nicer. Closing this issue.
